README.md

```diff
@@ -93,6 +93,12 @@ In case you ever need to regenerate the SSL certificate:
 
 If both these parameters are present, the node will also start service SSL traffic.
 
+### Agent
+
+```from v1.12.0```
+
+`--start-agent` start agent for cleanup and provisioning purposes 
```

Comment:
its actually not connected yet. agent is created either way. even when it will be connected, the default val is true so no need to worry about people forgetting. the rational is mainly to be able to shut it down easily with no boyar deployment in case things go south.